### DISCUSSION TITLE
Change routes: First login then main dashboard screen

### DISCUSSION THREAD

**Discussion by CarlosErnesto85** [primefaces/community#1281] [PrimeNG Templates] [answered]
Greetings community, I am new and I was able to include prime-ng to an empty project, but I would like to change the sequence: first load the login window then the layout dashboard, but it has a strange behavior, it always redirects to the login. Could someone guide me? thank you.

https://github.com/primefaces/community/assets/102236652/ed7fc40b-8bc3-444e-a5fb-6ffda57d24a3

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You need to create a new route containing your login page as the initial route. In the `app-routing.module.ts` file you'll see the code below

```
const routes: Routes = [
    {
        path: '',
        component: AppLayoutComponent,
        children: [
            {
                path: '',
                loadChildren: () =>
                    import(
                        './demo/components/dashboards/dashboards.module'
                    ).then((m) => m.DashboardsModule),
            }
        ],
    }
];
```

In here you need to replace the initial path which loads the dashboardsmodule with your module or page to load. It should look like this:

```
        path: '',
        component: AppLayoutComponent,
        children: [
            {
                path: '',
                data: { breadcrumb: 'Auth' },
                loadChildren: () =>
                    import('./demo/components/auth/auth.module').then(
                        (m) => m.AuthModule
                    ),
            }
        ]
```

And in the auth module or the module your login page is located

```
@NgModule({
    imports: [
        RouterModule.forChild([
            {
                path: '',
                loadChildren: () =>
                    import('./login/login.module').then((m) => m.LoginModule),
            },
        ]),
    ],
    exports: [RouterModule],
})
export class AuthRoutingModule {}
```

To learn and understand common routing tasks in angular please visit https://angular.dev/guide/routing

**Comment by CarlosErnesto85** [comment]
File: **app-routing.module.ts** modified

![image](https://github.com/primefaces/community/assets/102236652/55a891f3-7615-405d-a28f-fc999b3adbde)


File: **auth-routing.module.ts** modified

![image](https://github.com/primefaces/community/assets/102236652/8a0b29a7-7847-4682-b819-fe627b431877)


File: **login.component.html** modified
![image](https://github.com/primefaces/community/assets/102236652/1f420a09-e98b-404e-87c5-0c76a022aaec)

Video: 

https://github.com/primefaces/community/assets/102236652/8b12451c-9bdb-43b2-9e46-9a6c2a6048ba



Greetings, thank you very much for your time, I made the changes now if the login window loads with the initial window, but when I click on the login button it shows notfound, but when I enter the manual path, it goes to the login.
That could be happening?. Thank you
